Makefile

```diff
@@ -0,0 +1,13 @@
+## builds chain core with c dependencies, for development environments
+build-dev: build-lib
+	go install -tags 'localhost_auth http_ok reset' chain/cmd/cored
+
+## builds statically-linked c dependencies
+build-lib:
+	cd $(CHAIN)/vendor/github.com/google/snappy; mkdir build; cd build && cmake ../ && make
```

Comment:
Add a definition at the top of the Makefile: `SNAPPY = $(CHAIN)/vendor/github.com/google/snappy`. Then split this up into multiple lines:
```
    -mkdir $(SNAPPY)/build
    cd $(SNAPPY)/build && cmake ..
    cd $(SNAPPY)/build && $(MAKE)
```

(The leading hyphen means "ignore errors." The predefined variable `MAKE` does magic when doing recursive makes, see https://www.gnu.org/software/make/manual/make.html#MAKE-Variable.)